File: CT_BAEKJOON_1330.py
#https://www.acmicpc.net/problem/1330
#https://www.acmicpc.net/source/41978731

a ,b = input().split()

a = int(a)
b = int(b)



if a>b:
    print(">")
elif a<b:
    print("<")
elif a==b:
    print("==")

File: CT_BAEKJOON_1330.c
//https://www.acmicpc.net/problem/1330
//https://www.acmicpc.net/source/40065796

#include <stdio.h>
#pragma warning(disable:4996)
int main()
{
	int A , B , fir ,sec, th;
	scanf("%d %d", &A, &B);
	if (A > B) {
		printf(">");
	}
	else if (A < B) {
		printf("<");
	}
	else {
		printf("==");
	}
}	